100joon/피보나치.js

```diff
@@ -0,0 +1,16 @@
+const fs = require("fs");
+const input = fs.readFileSync("/dev/stdin").toString();
+
+let inputValue = Number(input);
```

Comment:
변하지 않는 변수는 const 처리해주기
```suggestion
const inputValue = Number(input);
```